packages/client/modules/email/components/SummaryEmail/MeetingSummaryEmail/RetroTopic.tsx

```diff
@@ -121,7 +121,10 @@ const RetroTopic = (props: Props) => {
 
   const {reflectionGroup, discussion, id: stageId} = stage
   const {commentCount, discussionSummary} = discussion
+  const validDiscussionSummary =
+    discussionSummary && discussionSummary !== OpenAIMagicWords.NO_SUMMARY_RESPONSE
   const {reflections, title, voteCount, topicSummary} = reflectionGroup!
+  const validTopicSummary = topicSummary && topicSummary !== OpenAIMagicWords.NO_SUMMARY_RESPONSE
```

Comment:
-1 I think the server should filter out any invalid summaries rather than checking if it says "No" on the client. 

Rather than the server returning "No", perhaps it could return `null`, so we just need to check if the summary exists?